### DISCUSSION TITLE
Ability to turn off the first search

### DISCUSSION THREAD

**Discussion by dkalpakchi** [searchkit/searchkit#1246] [Q&A] [answered]
Hi!

I need searchkit to not perform the search when loading for the first time. I've noticed that in previous versions of searchkit there was a handy boolean flag searchOnLoad that one could set, but now it seems to be gone?

Is there any way to do such a thing now?

Thanks in advance!

**Comment by joemcelroy** [comment, marked as answer]
Yeh so with Searchkit now relying on instantsearch for components, the way they mention advocate is to proxy the search method and add conditional logic when its the first initial request. 

They have a doc article about this here https://www.algolia.com/doc/guides/building-search-ui/going-further/conditional-requests/react-hooks/
